src/libs/actions/Report.ts

```diff
@@ -446,7 +446,7 @@ function addActions(reportID: string, text = '', file?: FileObject) {
     let commandName: typeof WRITE_COMMANDS.ADD_COMMENT | typeof WRITE_COMMANDS.ADD_ATTACHMENT | typeof WRITE_COMMANDS.ADD_TEXT_AND_ATTACHMENT = WRITE_COMMANDS.ADD_COMMENT;
 
     if (text && !file) {
-        const reportComment = ReportUtils.buildOptimisticAddCommentReportAction(text);
+        const reportComment = ReportUtils.buildOptimisticAddCommentReportAction(text, undefined, undefined, undefined, undefined, reportID);
```

Comment:
```suggestion
        const reportComment = ReportUtils.buildOptimisticAddCommentReportAction(text, undefined, undefined, 0, undefined, reportID);
```

Reply:
I think we need to pass `createdOffset` with 0 not undefined, because if it undefined `created` will be also undefined.
`created: [Date.now() + createdOffset](created: DateUtils.getDBTimeWithSkew(Date.now() + createdOffset))`